tests/test_cli.py

```diff
@@ -219,6 +218,22 @@ def test_output_filename_uncommon(
         result = self.run_cli([sample_pdf, "--output-filename", output_filename])
         result.assert_success()
 
+    ###
```

Comment:
nitpick: you can remove this comment

Reply:
I'll fix this up, I can go ahead and add the changes and then merge the pr